dashboard/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "dashboard",
+  "version": "0.0.1",
+  "scripts": {
+    "prod": "svelte-kit dev --port=3055 --host=0.0.0.0",
```

Comment:
Please remove the --port and --host command.
Caller need to customize this from outside

Reply:
as bespoke we only remove the port param otherwise the dashboard wont be accessible via docker